src/qutip_qtrl/pulsegen.py

```diff
@@ -1042,16 +1044,18 @@ def init_pulse(self, num_coeffs=None):
     #            self.guess_pulse = self.guess_pulsegen.gen_pulse()
     #        return self.guess_pulse
 
-    def init_coeffs(self, num_coeffs=None):
+    def init_coeffs(self, num_coeffs=None, init_param_vals=None):
         """
-        Generate the initial ceofficent values.
+        Generate or set the initial ceofficent values.
 
         Parameters
         ----------
         num_coeffs : integer
             Number of coefficients used for each basis function
             If given this overides the default and sets the attribute
             of the same name.
+        init_param_vals : float array[num_coeffs * num_basis_funcs]
```

Comment:
Would it be better to call this parameter `init_coeffs` since it is passed directly to `set_coeffs`?

Reply:
I think that does make sense